src/components/Branding/styled-components.js

```diff
@@ -3,12 +3,14 @@ import { color } from '../common-styled-components';
 
 export const BrandingContainer = styled.div`
   position: absolute;
+  z-index: 1;
   top: 8px;
   left: 0;
   width: 64px;
-  z-index: 1;
-  text-align: center;
   font-size: 28px;
+  text-align: center;
+  cursor: default;
```

Comment:
Good thinking, but it's unneeded since your cursor is already default if you disable user-select